Makefile

```
@@ -0,0 +1,38 @@
+CODE_FOLDERS := src
+TEST_FOLDERS := tests
+
+# Poetry
+
+install:
+	poetry install --no-root
+
+update:
+	poetry lock
+	poetry install --no-root
+
+# Test
+
+test:
+	pytest $(TEST_FOLDER) --cov=$(CODE_FOLDERS)
+
+# Lint
+
+format:
+	black . -S
```

Можно сразу добавить `ruff` (https://docs.astral.sh/ruff/)
```suggestion
	ruff check --fix .
	ruff format .
	black . -S
```